src/DssSpell.t.base.sol

```diff
@@ -3584,6 +3584,24 @@ contract DssSpellTestBase is Config, DssTest {
         }
     }
 
+    function _fixChronicleStaleness(address oracle) internal {
+        bytes32 pokeDataSlot = bytes32(uint256(4));
+        bytes32 pokeData     = vm.load(oracle, pokeDataSlot);
+        uint128 pokePrice    = uint128(bytes16(pokeData << 128));
+
+        uint32 expiresAt = 365 days * 100;
+
+        vm.store(oracle, pokeDataSlot, bytes32(uint256(expiresAt) << 128 | uint256(pokePrice)));
+    }
+
+    function _fixChronicleStaleness() internal {
```

Comment:
This function should probably be called in `setUp` in `DssSpellBaseTest`.
An alternative, to not mess with the base test, would be to add a `setUp` function in the main spell test suite that calls it:

```solidity
    // DssSpell.t.sol
    function setUp() {
        super.setUp();
        _fixChroniclesStaleness();
    }
```